src/components/user/user.service.ts

```diff
@@ -366,6 +373,49 @@ export class UserService {
       nodevar: 'user',
     });
 
+    // Update email
+    if (input.email) {
+      // Remove old emails and relations
+      await this.db
+        .query()
+        .match([
+          node('user', ['User', 'BaseNode'], { id: user.id }),
+          relation('out', 'oldRel', 'email', { active: true }),
+          node('email', 'EmailAddress:Property'),
+        ])
+        .delete('oldRel')
+        .delete('email')
```

Comment:
I think we just we the old relationships marked `active: false` since of deleting them. For history.